packages/manager/modules/pci/src/projects/project/data-processing/submit-job/translations/Messages_fr_FR.json

```diff
@@ -7,6 +7,8 @@
   "data_processing_submit_job_stepper_job_type_title": "Sélectionner un type de job",
   "data_processing_submit_job_stepper_sizing_title": "Dimensionnement du job",
   "data_processing_submit_job_stepper_configure_title": "Configurer votre job",
+  "data_processing_submit_job_stepper_submit_title": "Soumettre votre job",
+  "data_processing_submit_job_command_description": "La commande suivante vous permet de créer votre job grâce à <a class=\"oui-link_icon\" href=\"{{:: apiGuideUrl }}\" target=\"_blank\" rel=\"noopener\"> notre API <span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span> </a>.",
```

Comment:
Sorry I should send more recommendation. It's ok you have merge the content in one sentence.

But maybe you could have the whole text as the link? In order to avoid having HTML markups in content, which can be an hassle if we have to set tracking on the link.

Reply:
Hi @JeremyDec 
After some exchanges with the shareholders from Data Processing and CloudDB (as it is the same issue that we can find in https://github.com/ovh/manager/pull/5839#discussion_r759971841), we feel it would be confusing for the customers to have the whole text as the link. However, we decided to split this sentence into two different sentences on two lines with the second one being a link. This way, there is no split in the middle of the sentence and no HTML markups in content. Please feel free to tell me if there is still an issue.